store/tikv/oracle/oracles/pd.go

```diff
@@ -231,3 +231,32 @@ func (o *pdOracle) GetLowResolutionTimestampAsync(ctx context.Context, opt *orac
 		err: nil,
 	}
 }
+
+func (o *pdOracle) getStaleTimestamp(ctx context.Context, prevSecond int64) (uint64, error) {
+	ts, ok := o.getLastTS("")
```

Comment:
passing oracle.GlobalTxnScope directly.

Reply:
Thanks for your review, I agree with you.

Reply:
Thanks for your review, I agree with you.